src/components/SQForm/SQFormResetInitialValuesButton.tsx

```diff
@@ -13,14 +30,15 @@ function SQFormResetInitialValuesButton({
   tooltip = 'Reset Button',
   initialValuesObject = {},
   ...props
-}) {
+}: SQFormResetInitialValuesButtonProps): JSX.Element {
   const {isDialogOpen, openDialog, closeDialog} = useDialog();
-  const {values, resetForm} = useFormButton(isDisabled);
+  const {values, resetForm} = useFormButton(isDisabled, false);
```

Comment:
> Still getting error on values.

`useFormButton` now accepts a `Values` generic that will tell `useFormikContext` what shape the form's values are in. With that, the `values` that is returned from `useFormButton` will be typed to what is passed in as the generic. 
So, we need to pass in the type of the values object:
```suggestion
  const {values, resetForm} = useFormButton<typeof initialValuesObject>(isDisabled, false);
```
Then `values` will no longer need to be cast as `FormikValues`